operations/aws-cli-v2.yml

```diff
@@ -0,0 +1,19 @@
+- type: replace
+  path: /instance_groups/name=bosh/jobs/-
+  value: 
+    name: pre-start-script
+    release: os-conf
+    properties:
+      script: |
+        #!/bin/bash
```

Comment:
Is this a pretty standard way to do things in BOSH?

Reply:
It is a workaround for now, if we get time to make a full bosh release for this, or can find another donor release, we can do that.

We use os-conf release for a number of odds-n-ends installs